repmgr-action-standby.c

```diff
@@ -687,19 +689,18 @@ do_standby_clone(void)
 		exit(SUCCESS);
 	}
 
-	if (mode != barman)
```

Comment:
Won't this break backwards compatibility?

Reply:
Maybe we can or this with pg_backupapi. The other option would be to default the value of mode to pg_basebackup

Reply:
From reading the changes below this seems to be the case, meaning that we do set to pg_basebackup by default if barman not pg_backup_api is provided